docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -78,11 +92,13 @@ The response looks like:
 --------------------------------------------------
 <1> Whether the shard is assigned or unassigned
 <2> Reason for the shard originally becoming unassigned
-<3> List of node decisions about the shard
-<4> User-added attributes the node has
-<5> Final decision for whether the shard is allowed to be allocated to this node
-<6> Weight for how much the allocator would like to allocate the shard to this node
-<7> List of decisions factoring into final decision
+<3> Configured delay before the shard can be allocated
+<4> Remaining delay before the shard can be allocated
+<5> User-added attributes the node has
```

Comment:
`User-added attributes the node has` -> `The user-added attributes for the node`?